sbu-kalm/KALM
# Frame management

Tasks: 
- [x] Implemented frames & roles table view for existing frames based on converted frames.json file
- [x] Added delete + add frame modals
- [x] Removed active navbar selection <- Will raise issue with current logic of grabbing the currently selected navlink
- [x] Added mantine/form & notifications to package.json which will later be utilized to save modal values
- [x] Implemented Clean Frame table <- 2nd table will need to be updated with frame examples 

Future Implementations:
- [ ] Add Edit Modal
- [ ] Write to Frames.json after add, delete, edit

How to Test:
```
git checkout main
git pull
git checkout frame_management
cd client
npm start
```

See the below screens for Frame Management
Frame Management Table
<img width="1627" alt="image" src="https://github.com/sbu-kalm/KALM/assets/53535722/880124f5-7c9b-4598-bb3d-0974a8795adf">
Delete frame
<img width="1633" alt="image" src="https://github.com/sbu-kalm/KALM/assets/53535722/bf8c7bf1-8e25-4527-be69-f5e7ed6e113c">
Add a frame
<img width="1633" alt="image" src="https://github.com/sbu-kalm/KALM/assets/53535722/9512be29-26b6-4378-9528-e5ff9014e527">
